src/flask_container_scaffold/app_scaffold.py

```diff
@@ -0,0 +1,65 @@
+import os
+
+from flask import Flask
+
+from flask_container_scaffold.util import load_yaml
+
+
+class AppScaffold(object):
+
+    def __init__(self, app=None,
+                 name=__name__, config=None):
```

Comment:
The comment on line 37 suggests that config is only expected to be set in a testing environment, I wonder if a docstring might help makes this usage expectation explicit.

Reply:
Good idea, I should have a docstring here anyway.  I guess I didn't remember because I am thinking of the current implementation as pretty transitory.  Will remove 'test' from the comment below as well - originally, this was called test_config, but ended up being used for more than that.